### ISSUE TITLE
Changing Workspace Indicator Labels Breaks next-workspace

### ISSUE THREAD

**Issue by silentjay** [regolith-linux/regolith-desktop#513]
- Regolith Install Type:PPA
 - Regolith Version: 1.5
 - PPA url: [default: ppa:regolith-linux/release]
 - Host OS (for PPA): Ubuntu 20.04.1

Following the guide here: https://regolith-linux.org/docs/howto/remove-workspace-icons/ to change my workspace labels results in the `next-workspace` cmd (super + `) cmd breaking. 

If I run the cmd in the shell `/usr/share/i3xrocks/next-workspace --startnum 1` I get the error:
```
Traceback (most recent call last):
 File "/usr/share/i3xrocks/next-workspace", line 23, in <module>
    font = workspaces[0].name.split("'")[1]
IndexError: list index out of range
```

Here's my `/etc/regolith/styles/i3-wm`:
```
#define WORKSPACE_NAME(INDEX, FONT, COLOR, GLYPH) 
i3-wm.workspace.01.name: 1: Terminal
i3-wm.workspace.02.name: 2: Web
i3-wm.workspace.03.name: 3: Chat
```
I've swapped out i3bar for polybar, but I don't believe that's the issue here.

**Comment by moritzheiber** [comment]
@ferdinandyb I also believe [the documentation](https://regolith-linux.org/docs/howto/remove-workspace-icons/) needs updating?

**Comment by ferdinandyb** [comment]
This is already fixed in the [master](https://github.com/regolith-linux/regolith-i3xrocks-config/blob/master/scripts/next-workspace), I think it will go into 1.5.1 in a week or two hopefully.

On the other hand, although the documentation _should_ work, I think the preferred way to edit this would be in `.config/regolith/Xresources`, as that doesn't change a "system" config. but there you need to pass to formatting as well (at least I haven't been able to come up with any way to circumvent this):

e.g.:
```
i3-wm.workspace.01.name: 1:<span font_desc='JetBrains Mono Medium 13'> Terminal </span> 
```

**Comment by kgilmer** [comment]
Good call-out on the documentation, I'll fix that now @moritzheiber

**Comment by moritzheiber** [comment]
This has been shipped as a part of 1.5.1
